reactor-netty-core/build.gradle

```diff
@@ -54,6 +54,7 @@ dependencies {
 	compile "io.netty:netty-handler:$nettyVersion"
 	compile "io.netty:netty-handler-proxy:$nettyVersion"
 	compile "io.netty:netty-resolver-dns:$nettyVersion"
+	compile "io.netty:netty-resolver-dns-native-macos:$nettyVersion:osx-x86_64"
```

Comment:
should that os-specific artifact be included in the logic of `forceTransport` below?